backend/apid/middlewares/namespace.go

```diff
@@ -0,0 +1,34 @@
+package middlewares
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/sensu/sensu-go/backend/store"
+	"github.com/sensu/sensu-go/types"
+)
+
+const (
+	defaultNamespace = "default"
+)
+
+// Namespace retrieves the namespace passed as a query parameter and validate
+// its existence against the data store and then add them to the request context
```

Comment:
I'm looking at the Then method, but I don't see the store ever being queried.

Do we need the store?

Reply:
Excellent point! I went over the store code and we are now making sure the namespace exists with transactions before creating anything, so this middleware doesn't have any reason to exist. Let's remove it 🔥